openml/tasks/split.py

```diff
@@ -10,6 +10,14 @@
 
 
 class OpenMLSplit(object):
+    """OpenML Split object.
+
+       Parameters
+       ----------
+       name : int or str
+       description : str
+       split : int
```

Comment:
This seems to be incorrect. Shouldn't this be a dictionary?

Reply:
Yeah, it should be `dict`.